### PR TITLE
Fixed Focus in Search Stream

### PR DESCRIPTION
**Pull request by Ar9av**
- Stream maintains order when text is removed after entering in the search stream

- Added pytest test case for the same

### CODE REVIEW

**Comment by preetmishra** [comment]
@Ar9av Hey, thanks for your initiative to contribute to Zulip. It looks like your code is failing a linting test, `pycodestyle` specifically.

It is recommended to run all the linting tests along with `pytest` locally before you push your code for review (You can read more about it [here](https://github.com/zulip/zulip-terminal#development-tasks)).

However, if you would like to discuss why the linting test is failing, you can post your query here or in the [#zulip-terminal](https://chat.zulip.org/#narrow/stream/206-zulip-terminal) stream. :)

File: tests/ui/test_ui_tools.py [hunk 1]
```diff
@@ -415,6 +415,8 @@ def test_init(self, mocker, stream_view):
         ('FOO', ['FOO', 'FOOBAR', 'foo']),
         ('test', ['test here']),
         ('here', ['test here']),
+        ('', ['FOO', 'FOOBAR', 'foo', 'fan',
+            'boo', 'BOO', 'bar', 'test here'])
     ])
     def test_update_streams(self, mocker, stream_view, new_text, expected_log):
         stream_names = [
```
Comment (amanagr) [inline tests/ui/test_ui_tools.py]: @Ar9av I think you just need test if `log.set_focus` was called. No other change in needed I think.

**Comment by neiljp** [comment]
@Ar9av Since you've not referenced an issue, could you describe the bug this is fixing in a little more detail?

**Comment by neiljp** [comment]
@Ar9av Hey, just checking in with you on this; we'd love to have your contribution, but it'd be good to hear more from you regarding this :+1:

We did fix some bugs recently for the stream searching, so this may not be an issue any more, but please let us know :)

**Comment by neiljp** [comment]
@Ar9av Since we've not heard from you in a while I'm going to close this pull request. If you want to continue this work, you're welcome to do so, though please identify which problem you're trying to address up-front.

**Pull request closed without merging**